docs/src/SBP_Operators.md

```diff
@@ -0,0 +1,182 @@
+
+The framework has since been extended to encompass a much broader class of methods, making it a unifying principle in numerical analysis. This generality allows practitioners to choose the most appropriate discretization for their specific problem while maintaining the same theoretical guarantees and structure-preserving properties.
+
+[^LampertRanocha2024]:
+    Lampert, Ranocha (2024):
+    Structure-Preserving Numerical Methods for Two Nonlinear Systems of Dispersive Wave Equations
+    [arXiv: 2402.16669](https://arxiv.org/abs/2402.16669)
+
+## 2. Mathematical Foundation
+
+### The Core SBP Property
+
+The fundamental property that defines a first-derivative SBP operator is:
+
+```math
+MD + D^T M = t_R t_R^T - t_L t_L^T
+```
+
+where:
+
+- ``D`` is the derivative operator (matrix)
+- ``M`` is the symmetric, positive definite mass matrix
+- ``t_L = (1,0,...,0)^T`` and ``t_R = (0,...,0,1)^T`` extract boundary values
+
+This property is the discrete analog of integration by parts:
+
+```math
+\begin{aligned}
+\underbrace{ \boldsymbol{u}^T M D \boldsymbol{v} + \boldsymbol{u}^T D^T M \boldsymbol{v} }_{\displaystyle \approx \int_{x_{\min}}^{x_{\max}} u\, (\partial_x v) + \int_{x_{\min}}^{x_{\max}} (\partial_x u)\, v } 
+&= 
+\underbrace{ \boldsymbol{u}^T \boldsymbol{t}_R \boldsymbol{t}_R^T \boldsymbol{v} - \boldsymbol{u}^T \boldsymbol{t}_L \boldsymbol{t}_L^T \boldsymbol{v} }_{\displaystyle \approx u(x_{\max})\,v(x_{\max}) - u(x_{\min})\,v(x_{\min}) }.
+\end{aligned}
+```
+
+where for periodic SBP operators their property naturally simplifies to
+
+```math
+MD + D^T M = 0.
+```
+
+### Understanding the Mass Matrix
+
+The mass matrix ``M`` approximates the continuous inner product:
+
+```math
+\langle u, v \rangle_M = u^T M v \approx \int_{x_{min}}^{x_{max}} u(x) v(x) dx
+```
+
+For the approximation to be meaningful, we require:
+
+```math
+1^T M 1 = x_{max} - x_{min}
+```
+
+This ensures that the discrete inner product correctly integrates constants.
+
+### Understanding the Derivative Operator
+
+The derivative operator ``D`` approximates the spatial derivative of a function on a discrete grid. That is, for a smooth function ``u(x)``, we want the discrete expression ``D \boldsymbol{u}`` to approximate ``\partial_x u(x)`` at the grid points.
+
+A first-derivative SBP operator is said to be **$p$-th order accurate** if it satisfies:
+
+```math
+D\, \boldsymbol{x}^k = k\, \boldsymbol{x}^{k-1}, \quad \text{for all } k = 0, 1, ..., p,
+```
+
+where ``\boldsymbol{x}^k = (x_1^k, \dots, x_N^k)^T`` and the operations are performed pointwise. This means the discrete operator correctly differentiates monomials up to degree $p$. In particular, **consistency** requires at least zeroth-order accuracy, i.e.,
```

Comment:
```suggestion
where ``\boldsymbol{x}^k = (x_1^k, \dots, x_N^k)^T``. This means the discrete operator correctly differentiates monomials up to degree $p$. In particular, **consistency** requires at least zeroth-order accuracy, i.e.,
```
I think the only pointwise operation is the exponentiation, but if you already explicitly define that, we don't need to say again that the operation is performed pointwise (and otherwise, one could be confused with $$D\boldsymbol{x}$$ being performed pointwise.